src/app/auth/account-settings/account-settings.component.ts

```diff
@@ -13,13 +13,24 @@ import { CognitoUserFacade } from 'src/app/shared/models/auth/user';
 })
 export class AccountSettingsComponent implements OnInit {
   protected currentUser!: CognitoUserFacade | undefined;
+  subscribe: Subscription | undefined;
```

Comment:
```subscription``` instead of ```subscribe```
I think it can be:
subscription!: Subscription;

Reply:
Yes, but I didn't like it that way and the Eslint suggested that one to me